core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -38,8 +38,11 @@ class HttpWorkspaceManagerDAO(baseWorkspaceManagerUrl: String)(implicit val syst
     getWorkspaceApi(accessToken).deleteWorkspace(workspaceId)
   }
 
-  override def createDataReference(workspaceId: UUID, name: DataReferenceName, description: DataReferenceDescriptionField, referenceType: ReferenceTypeEnum, reference: DataRepoSnapshot, cloningInstructions: CloningInstructionsEnum, accessToken: OAuth2BearerToken): DataReferenceDescription = {
-    getWorkspaceApi(accessToken).createDataReference(new CreateDataReferenceRequestBody().name(name.value).description(description.value).referenceType(referenceType).reference(reference).cloningInstructions(cloningInstructions), workspaceId)
+  override def createDataReference(workspaceId: UUID, name: DataReferenceName, description: Option[DataReferenceDescriptionField], referenceType: ReferenceTypeEnum, reference: DataRepoSnapshot, cloningInstructions: CloningInstructionsEnum, accessToken: OAuth2BearerToken): DataReferenceDescription = {
+    val request = new CreateDataReferenceRequestBody().name(name.value).referenceType(referenceType).reference(reference).cloningInstructions(cloningInstructions)
+    description.map(d => request.description(d.value))
+
+    getWorkspaceApi(accessToken).createDataReference(request, workspaceId)
   }
```

Comment:
this method could accept a `NamedDataRepoSnapshot` as an argument, instead of `DataReferenceName` and `Option[DataReferenceDescriptionField]`. Optional - I wouldn't block the PR on that.

Reply:
I'm going to pass on this for now, it seems to add some complication to the logic (i.e. repeated snapshot UUID), and I think puzzling through that is a task for another PR. For now, I'm going to move ahead with this.